### ISSUE TITLE
DefaultNullRead of @Parsed does not work with enums

### ISSUE THREAD

**Issue by ammasca** [uniVocity/univocity-parsers#427]
Trying to parse a csv with missing values for some columns.
I need to set a default for missing column value to a field of type enum by using the 
  `@Parsed(defaultNullRead="myDefaultVal")`

It's working for other data types but not for enums. 
For enums I always get null value in field when column value is missing.

```java
  public enum CarType {
    SUV,
    LIMO
  }

  @Parsed(field = "carType", defaultNullRead = "SUV")
  private CarType carType;
```
Csv file:
```
id,carType,averageConsumtion
2,LIMO,15
1,,10
```

Tried also by initializing the field at definition but doesn't work either 
```java
  @Parsed(field = "carType")
  private CarType carType = CarType.SUV;
```

The only way I got this working is by annotating the setter and checking for null value in the body, but is kind of clumsy.

```java
  @Parsed(field = "carType")
  public void setCarType(CarType carType) {
    if(carType == null) {
      this.carType = CarType.SUV;
    } else {
      this.carType = carType;
    }
  }
```

**Comment by jbax** [comment]
You can solve the problem by adding a @EnumOptions annotation there. 

But that's a bug and I've fixed it and I'll release a 2.9.1-SNAPSHOT build in an hour from now which you can use to test.

With that updated version the original code you posted should work fine.

Thank you for using our parsers!

**Comment by jonenst** [comment]
Did this break custom converters on enum ? Looking at https://github.com/uniVocity/univocity-parsers/commit/500b9b053bec2853a#diff-b30623f01abb657ccdaf4b0c4a22c0f8407a485fc27ac969c3c34aec8fc09769R392 how do you disable the converter to the enum ? I'm migrating from 2.8.3 to 2.9.1

In my case, I had
```java
    public enum Season {
        SUMMER,
        WINTER
    }

    @FixedWidth(from = 43, to = 44)
    @Convert(conversionClass = SeasonConversion.class)
    private Season season;
```
```java
public class SeasonConversion extends ObjectConversion<Season> {

    @Override
    protected IeeeCdfTitle.Season fromString(String str) {
        char season = str.charAt(0);
        switch (season) {
            case 'S':
                return IeeeCdfTitle.Season.SUMMER;
            case 'W':
                return IeeeCdfTitle.Season.WINTER;
            default:
                throw new AssertionError("Unknown season: " + season);
        }
    }
```
so now I have 2 converters, my converter and then enumconverter.
Using @Parsed(applyDefaultConversion = false) doesn't do anything.

As a workaround I can make my converter return the strings that are needed by the EnumConverter instead of make my converter return the enum elements, but it seems wasteful: 

```java
//workaround, convert to strings for next converter
public class SeasonConversion extends ObjectConversion<String> {


    @Override
    protected String fromString(String str) {
        char season = str.charAt(0);
        switch (season) {
            case 'S':
                return IeeeCdfTitle.Season.SUMMER.name();
            case 'W':
                return IeeeCdfTitle.Season.WINTER.name();
        }
    }
```
